docs/cookbook/Translations with SiteAccess

```diff
@@ -0,0 +1,119 @@
+## Description
+
+Being new to eZ and combining the translated content with multiple site access successfully can be a little tricky to get you head around.
+
+To successfully achieve this for the most typical setups you will need all 3 of following items to be considered:
+1. Registering your translations in the database via the eZ admin.
+1. Creating at least two site accesses to deliver the right translated content at the right time in the `ezplatform.yml`.
```

Comment:
Same thing "site accesses" needs to be First Upper Cased, please. (I will not repeat that every time, so I consider that you will do it for every appearance of these words.)